roles/confluent.kafka_connect_replicator/templates/kafka-connect-replicator-log4j.properties.j2

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+log4j.rootLogger=INFO, replicatorAppender, stdout
+
+
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+
+#
+# The `%X{connector.context}` parameter in the layout includes connector-specific and task-specific information
+# in the log message, where appropriate. This makes it easier to identify those log messages that apply to a
+# specific connector. Simply add this parameter to the log layout configuration below to include the contextual information.
+#
+#log4j.appender.stdout.layout.ConversionPattern=[%d] %p %X{connector.context}%m (%c:%L)%n
+log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
+
+log4j.logger.org.apache.zookeeper=ERROR
+log4j.logger.org.I0Itec.zkclient=ERROR
+log4j.logger.org.reflections=ERROR
+
+log4j.appender.replicatorAppender=org.apache.log4j.RollingFileAppender
+log4j.appender.replicatorAppender.File={{kafka_connect_replicator.appender_log_path}}{{kafka_connect_replicator.appender_log_name}}
```

Comment:
It doesn't appear these vars are defined `appender_log_path` and `appender_log_name`
But I think you want to make use of the LOG_DIR env var here if possible

Reply:
I see that they are defined, sorry! Still the use of the env var is the direction we've gone w other log4j properties

Reply:
It looks like the way log4j is handled is inconsistent across the components.  For example, if you review Control Center and then review kafka connect, they have different approaches to creating/updating the log4j file.  It's also not clearly documented.  Can you document the differences then I can make a more informed choice on the best way to implement this from a consistency perspective.

Reply:
Good point! The log4j changes were laid out in this PR: https://github.com/confluentinc/cp-ansible/pull/411
My intention was to use the out of the box log4j files whenever possible. I investigated whenever the log4j files already used the RollingFileAppender. So in the case of c3 there is no need for cp-ansible to get involved

For connect, a log4j file does get shipped w the packages that does not use the Rolling file appender, so  I added a script that customizes it. The intention was to future proof cp-ansible and to make owning the log4j conf files no longer our responsibility.

Ksql is an example of having a hard time moving away from our own log4j template because of the fact that the log streaming logic is so complex so it turned out easier just to ship our own log4j template for that component.

Replicator is the only component that does not ship with a log4j conf file in the packages (is that correct)?

One standard thing youll see across the log4j files used by our components after deployment thru cp-ansible is the use of the LOG_DIR env var. This is how it looks in the ksql log4j conf, but its similar in the rest:
```
log4j.appender.kafka.File=${ksql.log.dir}/ksql-kafka.log
```

Reply:
Right.  However this is a bad approach as it prevents customization for the customer.  So the question is, do I implement a bad standard, or do I do what's better for the customer and keep it in a configurable state?  I've also filed a bug to have our log4j logging configured back to where we own the templates so we can make everything fully configurable by the end user.

Reply:
Users don't know how to write log4j files, you can search through the support portal and find probably 100s of tickets of them needing help on how to configure log4j, even with a mostly complete template.  There is even a KB on it, pretty sure I wrote the original iteration actually.

Reply:
Test this `${kafka.logs.dir}` for the directory, make sure to allow for customization of that dir

Reply:
probably want its own directory within /var/log/confluent like the others have 
```
[root@kafka_connect_replicator1 /]# ls /var/log/confluent/
ce-kafka-http-server  ce-kafka-rest  confluent-metadata-service  control-center  kafka-connect-replicator.log
```

Reply:
After some investigation the LOG_DIR env var did not work for me, i think what i have in replicator-dom will resolve this